help.html

```diff
@@ -7,6 +7,12 @@
     </head>
     <body>
         <h1> Help </h1>
+        <h2> Using external tool </h2>
+        If you wish to temporarily try out a save,
+        you can use the third party tool <a href="https://github.com/LEGOlord208/UndertaleSandbox">UndertaleSandbox</a>.
+        Please note that we are in <b>no way</b> related to them, or any of their actions.
+        
+        If you wish to learn the manual way of installing, carry on with the tutorial!
         <h2> Getting Started </h2>
```

Comment:
I'm fine with suggesting other tools, but please make a separate pull request for unrelated changes.

I'll accept it this time though.